src/if.c

```
@@ -58,121 +67,178 @@
+#endif
+
+	return 1;
+}
+
+int
+if_link_match_index(const struct sockaddr *sa, unsigned int if_index)
+{
+	if (sa_is_link(sa) != 1) {
+		errno = EINVAL;
+		return -1;
+	}
```

Cppcheck (2.20.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)